server/serverAPI/controller/ShoppingCartController.ts

```diff
@@ -0,0 +1,248 @@
+import { ObjectID } from "bson";
```

Comment:
I think ShoppingList might be a more appropriate name than ShoppingCart, since we don't support actually purchasing anything